apps/weather/app.js

```diff
@@ -57,7 +57,7 @@ function draw() {
   layout.windUnit.label = wind[2] + " " + (current.wrose||'').toUpperCase();
   layout.cond.label = current.txt.charAt(0).toUpperCase()+(current.txt||'').slice(1);
   layout.loc.label = current.loc;
-  layout.updateTime.label = `${formatDuration(Date.now() - current.time)} ago`;
+  layout.updateTime.label = `${formatDuration(Date.now() - current.time)} ago`; // How to autotranslate this and similar?
```

Comment:
This should ideally use the `locale` module? If duration isn't handled then maybe we should add a new method to it for that... But right now with the tags you have minute/day/etc will get converted on